modules/mediawiki/files/robots.php

```diff
@@ -24,6 +28,37 @@
 echo "User-Agent: SemrushBot" . "\r\n";
 echo "Disallow: /" . "\r\n\n";
 
-# Dynamic sitemap url
-echo "# Dynamic sitemap url" . "\r\n";
-echo "Sitemap: https://$_SERVER[HTTP_HOST]/wiki/Special:RedirectSitemap" . "\r\n\n";
+if ( isset( $databasesArray['combi'] ) && $databasesArray['combi'] ) {
```

Comment:
Could avoid needing to do an isset by defining a blank array of [ 'combi' => [] ];